packages/babel-plugin-transform-modules-amd/test/fixtures/regression/16999/plugin.cjs

```diff
@@ -0,0 +1,13 @@
+module.exports = function plugin({ types: t }) {
+  let bool = false;
+  return {
+    visitor: {
+      Identifier(path) {
+        if (!bool) {
+          bool = true;
+          path.parentPath.parentPath.pushContainer("body", t.functionDeclaration(t.identifier("fn"), [t.identifier("x")], t.blockStatement([])));
```

Comment:
Can you add some assertions to this test so that it will fail without the changes in the amd transform? Currently this testing will be passing given the traverse change.